### ISSUE TITLE
Readme example code - subscribed check always returns false

### ISSUE THREAD

**Issue by eroelofsen** [laravel/cashier-mollie#13]
First of all: Thanks for this project.

I found that the example code in the readme which checks if a user is already subscribed always returns false as parameter "plan" should be the second argument instead of the first.

Current Readme.md
```
        public function __invoke(string $plan)
        {
            $user = Auth::user();
    
            if(!$user->subscribed($plan)) {
    
                $name = ucfirst($plan) . ' membership';
```

Suggested solution
```
        public function __invoke(string $plan)
        {
            $user = Auth::user();

            $name = ucfirst($plan) . ' membership';

            if(!$user->subscribed($name, $plan)) {
    
```

Billable.php
```
    /**
     * Determine if the billable model has a given subscription.
     *
     * @param  string  $subscription
     * @param  string|null  $plan
     * @return bool
     */
    public function subscribed($subscription = 'default', $plan = null)
```

**Comment by sandervanhooft** [comment]
Seems you're right @eroelofsen ! Can you submit a PR?

**Comment by sandervanhooft** [comment]
Fixed in PR #20 . Thanks @eroelofsen !
